### DISCUSSION TITLE
Default Route to new dashboards - Changes every time i login to Kibana

### DISCUSSION THREAD

**Discussion by LennyDK** [Security-Onion-Solutions/securityonion#3956] [Unsupported Versions]
Hi There
I am running 2.3.40
Question about how to make the Default Route work after log-in log-out in Kibana.

I have made a lot of new dashboards to work with Winlogbeat and Sysmon.
But when i change the "Default Route" in the web-gui to witch Dashboards i want it to load when i log in, it changes back to the Default Security Onion Dashboards when i log-on the next time.

What am i missing here ? Is it some form of user rights ? ore do it have to be set in salt/kibana/etc/config.json ???

Best Regards 
Lenny

**Comment by palichx** [comment]
i think this is issue https://github.com/Security-Onion-Solutions/securityonion/issues/3910

workaround:
try to change it in /opt/so/conf/kibana/saved_objects.ndjson.template

{"attributes":{"buildNum":33984,"dashboard:defaultDarkTheme":true,"defaultIndex":"XX","defaultRoute":"/app/dashboards#/view/YOUR_ID

**Comment by LennyDK** [comment]
Thanks 

Venlig hilsen / Best Regards
Lenny Hansson
*****************************
Web: https://networkforensic.dk
*****************************
E-mail: ***@***.***
Key ID: 5189 6702 3A0F 4C07 4AF5 5619 B6E0 D375 F0CB F2E8
*****************************
